### ISSUE TITLE
regional_indicator is not returning expected value

### ISSUE THREAD

**Issue by devsnek** [edm00se/emoji-transmogrifier#19]
### Expected Behavior

giving `regional_indicator_<letter>` should return the unicode for that
### Actual Behavior

returns null
### Steps to Reproduce

getUnicode('regional_indicator_<letter here>')
### Screenshots

![screenshot](http://i.imgur.com/qJGtjCT.png)

**Comment by edm00se** [comment]
Thanks for submitting the issue. I have a two-part response, with some explanation.

TL;DR:
- this module _could_/_may_/_probably-when-I-have-the-time_ evolve to include support for compatible country flag codes in the style of `flag-us`, `flag-ca`, in the style of Slack's emoji short code naming convention (with the two-letter country code pairs specifically)
- a full per-letter implementation of `regional_indicator_<X>` (single letters) is unlikely, unless someone submits [a decent PR](https://github.com/edm00se/emoji-transmogrifier/pull/new/master) (such a pull request would be welcome)

Longer version:
This project originated with the intention of interacting with the [gemoji](https://github.com/github/gemoji)/[jemoji](https://github.com/jekyll/jemoji) style short codes for conversion of values to images for an off-line capable generation of an eBook format of a copy of a [GitHub Pages](https://pages.github.com/) blog (via [gitbook-cli](https://github.com/GitbookIO/gitbook-cli)).

This means the scope was the available emoji short code names as found in gemoji/jemoji; specifically [what GitHub does via its API](https://api.github.com/emojis) (which this module is basically a wrapper for). This doesn't make this a "won't-fix" issue, but depending on what you're looking for it to accomplish, it may or may not be a perfect fit. I've added the `flag-XX` style names under the [Road Map in the Read Me](https://github.com/edm00se/emoji-transmogrifier#road-map) to reflect this as a planned feature addition.

Again, if you're looking to submit a PR to accomplish this, you are more than welcome to do so, otherwise I'll be taking the shorter path, when I get around to implementing it.

**Comment by edm00se** [comment]
Closing in favor of #20, which is specifically the feature request mentioned.
